src/main.cpp

```diff
@@ -208,6 +229,21 @@ int main(int argc, char * argv[]) {
 
     MixxxApplication app(argc, argv);
 
+#if defined(Q_OS_WIN)
+    // The Mixxx style is based on Qt's WindowsVista style
+    QApplication::setStyle("windowsvista");
+#endif
```

Comment:
I have just double checked on Linux using QT 6.2.4. and confirmed that you can move this block above `MixxxApplication app(argc, argv);` and remove `applyStyleOverride(&args);`

Reply:
I tested this, it does not work on my system, MixxxApplication app(argc, argv) overrides the style every time. It works neither with Qt6.5 nor with Qt6.8.